src/Illuminate/Support/Env.php

```diff
@@ -75,26 +75,17 @@ public static function get($key, $default = null)
     {
         return Option::fromValue(static::getRepository()->get($key))
             ->map(function ($value) {
-                switch (strtolower($value)) {
-                    case 'true':
-                    case '(true)':
-                        return true;
-                    case 'false':
-                    case '(false)':
-                        return false;
-                    case 'empty':
-                    case '(empty)':
-                        return '';
-                    case 'null':
-                    case '(null)':
-                        return;
-                }
+                $map = ['true' => true, 'false' => false, 'empty' => '', 'null' => null];
+
+                $key = strtolower(trim($value, '()'));
```

Comment:
This is actually not exactly the same as it will return `true` for all the following:

- `(true`
- `)true(`
- `))((true`